requirements/local.txt

```diff
@@ -1,4 +1,6 @@
 -r test.txt
+aiobotocore
+cached-property
```

Comment:
I think this is installed by `pip install -e .` (see https://loafer.readthedocs.io/en/latest/development/installation.html)

Reply:
My bad! I fix it